### PR TITLE
Update String.php

### PR DESCRIPTION
**Pull request by bureX**
Small typo ;)

### CODE REVIEW

File: String.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
  *     * Neither the name of the Hoa nor the names of its contributors may be
  *       used to endorse or promote products derived from this software without
  *       specific prior written permission.
- *
+ *p
```
Comment (Hywan) [inline String.php]: Please, remove this one :-).

Comment (bureX) [inline String.php]: Mental note: I'm a fat-fingered idiot. :)

**Comment by Hywan** [comment]
Hey thanks :-). Could you rebase your 2 patches into a single one please? Do you know how to do that?

**Comment by bureX** [comment]
It would be awesome if you could give me a hint, either via command line or via Atlassian SourceTree. Either way's fine.... I'm kinda lost on how to merge/rebase two patches into one. :(

**Comment by Hywan** [comment]
First, make sure the `master` is up-to-date:
```sh
$ git checkout master
$ git pull origin
```
then, go back to your branch:
```sh
$ git checkout patch-1
```
(It looks like a standard name for a patch from the Github interface, isn't it?)
Then, first, let put all your branch above the `master`:
```sh
$ git rebase master
```
Now, the real rebase of your patches:
```sh
$ git rebase -i 85c9d1e
// in your $EDITOR
pick 908335a Update String.php
pick 548a8b9 Update String.php
```
Change it for:
```
pick 908335a Update String.php
fixup 548a8b9 Update String.php
```
then save and quit.

I am doing it for you right now :-).

**Comment by Hywan** [comment]
Closed via https://github.com/hoaproject/String/commit/740e27e05e4f3d148daa359e7d074ed1853e0c03.

**Comment by Hywan** [comment]
I need your first and lastname for the [`CONTRIBUTOR.md` file](https://github.com/hoaproject/Central/blob/master/CONTRIBUTOR.md) (and [the website](http://hoa-project.net/Community.html#Contributors)) if you would like.

**Comment by bureX** [comment]
Nah, that's fine.
When I actually contribute something (and don't screw myself up in the process), I'll let you know :D

**Comment by Hywan** [comment]
@bureX First and lastname are not mandatory: https://github.com/hoaproject/Central/commit/904112e0adc02a1f5d610c5d9d27de0b8334737d ;-). Welcome on board. Waiting for awesome contributions! :metal:

**Pull request closed without merging**